### DISCUSSION TITLE
Blog post: New functions, gradients, and hues in CSS colors (Level 4)

### DISCUSSION THREAD

**Discussion by dipikabh** [mdn/mdn-community#377] [Content]
All feedback and comments related to: https://developer.mozilla.org/en-US/blog/css-color-module-level-4/

**Comment by fwqaaq** [comment]
Have you considered the idea of the localizing the blog? I think the blog is great, and developers from different parts of the world would benefits from it.

**Reply by schalkneethling** [reply]
Hey @fwqaaq, we just added [an announcement](https://github.com/orgs/mdn/discussions/379) answering questions like the one you raised here.

**Reply by fwqaaq** [reply]
Oh, thank you very much

**Comment by myakura** [comment]
> As of CSS Colors Module Level 3, you no longer need to use `rgba()` to define an RGB color with an alpha channel because `rgb()` supports alpha values separated by a slash (`/`) as demonstrated below.
https://developer.mozilla.org/en-US/blog/css-color-module-level-4/#defining_rgb_colors_in_css

I think that's Level 4 syntax update, not Level 3.

**Reply by bsmth** [reply]
Well-spotted, I'll update this shortly - thanks for raising 🙌🏻
